src/Attachment.tsx

```diff
@@ -267,12 +267,21 @@ export const AttachmentView = (props: {
                     receiptCardBuilder.addTextBlock(item.price, { horizontalAlignment: 'right' }, columns3[1]);
                 }
             });
-            const taxCol = receiptCardBuilder.addColumnSet([75, 25]);
-            receiptCardBuilder.addTextBlock('Tax', { size: "medium", weight: "bolder" }, taxCol[0]);
-            receiptCardBuilder.addTextBlock(attachment.content.tax, { horizontalAlignment: 'right' }, taxCol[1]);
-            const totalCol = receiptCardBuilder.addColumnSet([75, 25]);
-            receiptCardBuilder.addTextBlock('Total', { size: "medium", weight: "bolder" }, totalCol[0]);
-            receiptCardBuilder.addTextBlock(attachment.content.total, { horizontalAlignment: 'right', size: "medium", weight: "bolder" }, totalCol[1]);
+            if(attachment.content.vat){
+                const vatCol = receiptCardBuilder.addColumnSet([75, 25]);
+                receiptCardBuilder.addTextBlock('Vat', { size: "medium", weight: "bolder" }, vatCol[0]);
+                receiptCardBuilder.addTextBlock(attachment.content.vat, { horizontalAlignment: 'right' }, vatCol[1]);
+            }
+            if(attachment.content.tax){
```

Comment:
carefull. Having tax === 0 is a total valid usecase. But in javascript 0 is falsy, so it would not render here.

See my PR: https://github.com/Microsoft/BotFramework-WebChat/pull/633